source/retryable-writes/tests/README.rst

```diff
@@ -337,3 +342,5 @@ Changelog
              which are now expressed within ``runOn`` elements.
 
              Add test-level ``useMultipleMongoses`` field.
+
+:2020-06-12: Add the default value for heartbeatFrequencyMS.
```

Comment:
Per my [comment](https://github.com/mongodb/specifications/pull/820/files#r458261818) in the transactions README, this can be removed.